packages/cursorless-engine/src/actions/ParseTree.ts

```diff
@@ -0,0 +1,121 @@
+import { FlashStyle, Range, TextDocument } from "@cursorless/common";
+import * as path from "node:path";
+import type { Tree, TreeCursor } from "web-tree-sitter";
+import type { TreeSitter } from "..";
+import { ide } from "../singletons/ide.singleton";
+import type { Target } from "../typings/target.types";
+import { flashTargets } from "../util/targetUtils";
+import type { ActionReturnValue } from "./actions.types";
+
+export default class ParseTree {
```

Comment:
It's a bit awkward to have an action name which is a noun. I think it's ok for a spoken form, but for the name of the class and for our action ID, I'd prefer a verb, eg "show parse tree". Also makes it easier to grep for

Reply:
Sounds reasonable

Reply:
Done